tests/distances/braun_blanquet.py

```diff
@@ -6,23 +6,25 @@
     braun_blanquet_binary_distance,
     braun_blanquet_binary_similarity,
 )
-from tests.distances.utils import assert_similarity_and_distance_values
+from tests.distances.utils import assert_distance_values, assert_similarity_values
 
 
-def _get_values() -> list[tuple[list[int], list[int], str, float]]:
+def _get_values() -> list[tuple[list[int], list[int], str, float, float]]:
     return [
-        ([1, 0, 0], [0, 1, 1], "==", 0.0),
-        ([1, 0, 0], [0, 0, 0], "==", 0.0),
-        ([0, 0, 0], [0, 0, 0], "==", 1.0),
-        ([1, 0, 0], [1, 0, 0], "==", 1.0),
-        ([1, 1, 1], [1, 1, 1], "==", 1.0),
-        ([1, 0, 0, 0], [1, 1, 1, 1], "<", 0.5),
-        ([1, 1, 1, 0], [1, 1, 1, 1], ">", 0.5),
+        ([1, 0, 0], [0, 1, 1], "==", 0.0, 1.0),
```

Comment:
It would be useful to also add the comment about what those things mean above the list, i.e.:
```
def _get_values() -> ...:
    # vec_a, vec_b, comparison, similarity, distance
    return [
    ...
```